### ISSUE TITLE
Add support for prefix location (start/end)

### ISSUE THREAD

**Issue by silentbugs** [nosir/cleave.js#520]
Some currencies (such as EURO) are not prefixed with the currency sign, but it is added at the end. Example:

$50.51
50,51€

It would be great to have an option to indicate whether we'd like to have a prefix or 'postfix'.

**Comment by eatbuildplay** [comment]
Agreed this would be useful, in my experience the term is usually "suffix". I'd suggest just making "suffix" a separate option, in the event there is a need for both prefix and suffix in some situations.

**Comment by DvarfInkviz** [comment]
Are there any changes on this issue?
